cluster_tools/cluster_tools/schedulers/cluster_executor.py

```diff
@@ -615,11 +616,16 @@ def shutdown(self, wait: bool = True, cancel_futures: bool = True) -> None:
         self.wait_thread.stop()
         self.wait_thread.join()
 
+        self.clean_up()
+
+    def clean_up(self) -> None:
         for file_to_clean_up in self.files_to_clean_up:
             try:
                 os.unlink(file_to_clean_up)
-            except OSError:  # noqa: PERF203 `try`-`except` within a loop incurs performance overhead
-                pass
+            except OSError as exc:  # noqa: PERF203 `try`-`except` within a loop incurs performance overhead
+                logging.warning(
```

Comment:
I didn't find out yet why some successful runs didn't properly do the clean up. The added logging might help us in production to investigate this further.